src/main/java/org/myrobotlab/service/InMoov.java

```diff
@@ -154,9 +161,39 @@ public InMoov(String n) {
   // ---------------------------------------------------------------
 
   public void attach(Attachable attachable) {
+    //opencv
     if (attachable instanceof OpenCV) {
       opencv = (OpenCV) attachable;
+      subscribe(opencv.getName(), "publishClassification");
+      //SpeechRecognizer
+    } else if (attachable instanceof SpeechRecognizer) {
+      if (attachable.getClass().getSimpleName().equals("WebkitSpeechRecognition")) {
+        WebGui webgui = (WebGui) Runtime.create("webgui", "WebGui");
```

Comment:
you should not be changing the parameters of the webgui here nor should you be creating it.  It's already created if it's being passed into this method.  Please remove.